src/Oscoin/Protocol/Sync.hs

```diff
@@ -1,9 +1,14 @@
+-- | Syncing nodes in oscoin
+--
+-- Based on the spec available at https://hackmd.io/2cPkrWTjTIWo2EmHZpPIQw
```

Comment:
This is not permanent, I'd open a separate PR so we can merge it into the repo perhaps.

Reply:
Yeah. I have been thinking about it, and I'd like to experiment with post-processing in hackmd so perhaps we can render all those nice graphs & latex into a `.pdf` (or similar). In the meantime I have opened an issue to track that (cfr. #597 ). Do you think we can tackle this separately?

Reply:
Sure :+1: